src/stratis_cli/_errors.py

```diff
@@ -399,6 +399,12 @@ class StratisCliEnvironmentError(StratisCliError):
     """
 
 
+class StratisCliTestRunError(StratisCliError):
```

Comment:
You'll want to define this class in the tests module not in the src. Probably the best place is in _misc.py, along with the definition of ```TEST_RUNNER```.

Reply:
On a simple move, I'm getting a few lint errors:

- undefined-variable (for StratisCliError; should I choose something else?)
- too-few-public-methods (0/2)

Reply:
I think I got it: change "StratisCliError" to "Exception".

Reply:
Resolved.